src/osa/utils/cliopts.py

```diff
@@ -354,6 +354,7 @@ def provprocessparsing():
     options.configfile = opts.config.resolve()
     options.filter = opts.filter
     options.quit = opts.quit
+    options.no_dl2 = opts.no_dl2
```

Comment:
I think `no_dl2` is not defined in opts (from `provprocess_argparser`). There should be a `--no-dl2` option defined there as well. Although it might work if defined before by `closer` script, which it calls `provprocess`. In any case, I would add the cmd line option for provprocess and use it in its call